packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -20,18 +23,41 @@ export function Service({
   location,
   lastAccessTimeFormatted,
   canSignOut,
+  isCurrentSession,
+  clientId,
+  deviceId,
+  refreshTokenId,
+  sessionTokenId,
 }: {
   name: string;
   deviceType: string | null;
+  deviceId: string | null;
   location: DeviceLocation;
   lastAccessTimeFormatted: string;
   canSignOut: boolean;
+  isCurrentSession: boolean;
+  clientId: string;
+  sessionTokenId: string | null;
+  refreshTokenId: string | null;
```

Comment:
There's a `hexstring` type alias that we could use for some of these.

Reply:
Thanks, dodging that for now since the underlying types (src/models/Account.ts, also fxa-graphql-api/src/lib/resolvers/types/attachedClient.ts) use `string | null` and `string`, respectively